examples_tests/06.MeshLoaders/main.cpp

```diff
@@ -54,14 +54,13 @@ int main()
     asset::ICPUDescriptorSetLayout* ds1layout = mesh_raw->getMeshBuffer(0u)->getPipeline()->getLayout()->getDescriptorSetLayout(1u);
     //pipelines attached to meshbuffers from OBJ loader has DS1 layout being "irr/builtin/ds_layout/default_ds1_layout"
     //that is designed for use with "irr/builtin/ds/default_ds1", so let's use it!
```

Comment:
we should be doing this programmatically, not from **magical** convention knowledge that desc-set 1 is being used with this particular layout, and in that layout there is a UBO that requires that particular struct (still useful to have the struct in C++ for power-user).

Would be good to build it from pipeline metadata.